src/sdc11073/pmtypes.py

```diff
@@ -1374,3 +1373,40 @@ class PatientType(StringEnum):
     INFANT = 'Inf'
     NEONATAL = 'Neo'
     OTHER = 'Oth'
+
+
+class CalibrationResult(PropertyBasedPMType):
+    Code = cp.SubElementProperty([namespaces.domTag('Code')], valueClass=CodedValue)
```

Comment:
For anonymous XML structures the naming convention is to use a T_ prefix. This tells that this is not a named type in BICEPS.

Reply:
done